test/cases/ddl_test.rb

```diff
@@ -270,26 +273,29 @@ def test_create_spatial_column_default_value_geographic
       assert_equal 4326, col.srid
       assert_equal "0101000020E610000000000000000000000000000000000000", col.default
       assert_equal({ type: "st_point", srid: 4326, geographic: true }, col.limit)
-      klass.connection.drop_table(:spatial_models)
+      klass.lease_connection.drop_table(:spatial_models)
       assert_equal 0, count_geography_columns
     end
 
     def test_no_query_spatial_column_info
-      klass.connection.create_table(:spatial_models, force: true) do |t|
+      klass.lease_connection.create_table(:spatial_models, force: true) do |t|
         t.string "name"
       end
       klass.reset_column_information
-      # `all` queries column info from the database - it should not be called when klass.columns is called
-      ActiveRecord::ConnectionAdapters::PostGIS::SpatialColumnInfo.any_instance.expects(:all).never
-      # first column is id, second is name
-      refute klass.columns[1].spatial?
-      assert_nil klass.columns[1].has_z
+
+      # `SpatialColumnInfo#all` queries column info from the database.
+      # It should not be called when klass.columns is called
+      assert_queries_count(0) do
+        # first column is id, second is name
+        refute klass.columns[1].spatial?
+        assert_nil klass.columns[1].has_z
+      end
     end
 
     # Ensure that null contraints info is getting captured like the
```

Comment:
is it worth to fix the typo here?

```suggestion
    # Ensure that null constraints info is getting captured like the
```